pkg/apis/logstash/v1alpha1/name.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package v1alpha1
+
+import (
+	common_name "github.com/elastic/cloud-on-k8s/v2/pkg/controller/common/name"
+)
+
+const (
+	defaultServiceSuffix = "default"
```

Comment:
What do you think about using the name "api" rather than "default", to reflect that this service is for the Logstash API? So this would show up as `logstash-sample-ls-api`, rather than `logstash-sample-ls-default`?